internal/scheduler/context/context.go

```diff
@@ -34,6 +35,11 @@ type SchedulingContext struct {
 	PriorityClasses map[string]configuration.PriorityClass
 	// Default priority class.
 	DefaultPriorityClass string
+	// Determines how fairness is computed.
+	FairnessType configuration.FairnessType
+	// Used to convert one resource into another when computing fair share.
+	// Only applies to DominantResourceFairness.
+	FairnessResourceMappingBySourceResource map[string]configuration.ResourceMapping
```

Comment:
This doesn't seem to be used yet.

Also, it's not clear to me why resource mapping would only apply to `DominantResourceFairness`; the fractional GPU example you gave above seems like it also makes sense in the case of asset fairness.